liquibase-standard/src/main/java/liquibase/database/AbstractJdbcDatabase.java

```diff
@@ -946,9 +946,20 @@ protected boolean mustQuoteObjectName(String objectName, Class<? extends Databas
         if (isCatalogOrSchemaType(objectType) && preserveCaseIfRequested() == CatalogAndSchema.CatalogAndSchemaCase.ORIGINAL_CASE) {
             return true;
         }
+        addMySQLReservedWordIfApplicable("MANUAL");
```

Comment:
Instead of adding it inside `mustQuoteObjectName `, I think it's better to add a new method to Database interface and call it where it's required. The problem of adding it here is that it may produce side effects to `mustQuoteObjectName ` - you call a method to check quotes and gets something added to the reserved words.

Reply:
Hey Filipe, I agree with you that having this logic in `mustQuoteObjectName()` method was not in the right place. I was looking for options for a new method, but I thought moving this logic `AbstractjdbcDatabase.addReservedWords()` was good enough, so I moved it there. 

Could you please have a look at it again and please let me know if you think this is not the right approach?

Thanks,
Daniel.